doc/rst/developer/bestPractices/buildingdocs.rst

```diff
@@ -84,8 +88,12 @@ Examples include:
 
 
 
-Linking between files
----------------------
+Linking within and between files
+--------------------------------
+
+Linking within the same file
+ - Create a label right before the target section heading, ``.. _section-label:``.
+ - Use ``:ref:`section-label``` elsewhere in the same document.
```

Comment:
AFAIK this strategy allows also linking between files - these section labels are global

Reply:
I simplified this a bit by not differentiating the within and between file pieces.